### PR TITLE
add detail cards, pagination and goToDetails button

### PR DESCRIPTION
**Pull request by OoiYueYing**

### CODE REVIEW

File: src/component/DetailCard.jsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+const DetailCard = ({
+  title,
+  location,
+  description,
+  price,
+  imageLink,
+  /* eslint-disable-next-line */
+} = {}) => {
+  return title === undefined ? (
```
Comment (ooihuiying) [inline src/component/DetailCard.jsx:13]: A neater way is to write **return !title**

File: src/component/Pagination.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Pagination } from 'semantic-ui-react';
+
+const PaginationComponent = ({ totalPage }) => (
```
Comment (ooihuiying) [inline src/component/Pagination.jsx:4]: totalPages would be a more standardised way of naming

File: src/pages/Details.jsx [hunk 3]
```diff
@@ -1,9 +1,98 @@
...
+    const details = await mockSearchDetails(search);
+    const result = [];
+    details.forEach((detail) => {
+      result.push({
+        title: detail.title,
+        location: detail.location,
+        description: detail.description,
+        imageLink: detail.images[0],
+        /* eslint-disable-next-line */
+        price: detail.price.reduce((acc, { pricing }) => {
```
Comment (ooihuiying) [inline src/pages/Details.jsx:22]: wah reduce siol

File: package.json [hunk 4]
```diff
@@ -6,11 +6,14 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
+    "bootstrap": "^4.5.0",
     "react": "^16.13.1",
+    "react-bootstrap": "^1.0.1",
```
Comment (denzelchung) [inline package.json:11]: Bootstrap isn't used anymore right?

**Comment by denzelchung** [comment]
Is it possible to remove all the `/* eslint-disable-next-line */`

File: src/component/DetailCard.jsx (hunk 1 above)
Comment (OoiYueYing) [reply src/component/DetailCard.jsx:13]: got it!

File: src/component/Pagination.jsx (hunk 2 above)
Comment (OoiYueYing) [reply src/component/Pagination.jsx:4]: got it

File: package.json (hunk 4 above)
Comment (OoiYueYing) [reply package.json:11]: yup! i'll remove this

**Pull request merged**